src/openkit-dotnet-uwp/openkit-dotnet-uwp.csproj

```diff
@@ -0,0 +1,128 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If possible please also provide a Coverage configuration.

This is basically a Release configuration with Full Program database, so that coverage information can be obtained.